docs/getting-started/my-account/README.md

```diff
@@ -41,7 +41,7 @@ Enter your new password and confirm it a second time.
 
 Press the blue **Save** button in order to confirm the password changes.
 
-![change password](1572880507527.png)
+![my-account-change-password](Bildschirmfoto 2022-03-02 um 16.10.17.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.